tests/test_tifffile.py

```diff
@@ -3,14 +3,15 @@
 
 import os
 import datetime
-
 import numpy as np
 
 from pytest import raises
 from imageio.testing import run_tests_if_main, get_test_dir, need_internet
 from imageio.core import get_remote_file
-
 import imageio
+import imageio as iio
```

Comment:
I would pick one...

Reply:
> I would pick one...

Yep, I agree @mkcor and you are very correct in pointing this out.

I will move the entire test suite to `import imageio as iio` once I go through and do a test-styling PR. In the current suite, some files use `import imageio`, others use `import imageio as iio` and some use both. The change here actually doesn't worsen the situation overall 😅 .

There are a few other test-related things on my bucket list (serve standard images via a pytest-fixture, ensure that plugins are enforced for plugin-specific tests, etc.) so I think it makes sense to tread all of them in a dedicated PR that cleans up the test suite ... maybe we can even improve coverage in the process (wishful thinking™).

So yes, will be fixed. Unfortunately not in this PR, but in a future one instead. On the bright side, this PR will merge once the tests all pass :)